src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.bar {
+  font-weight: 900;
+  font-family: inherit;
+  display: block;
+  width: 100%;
+  box-shadow: 0 1px 8px #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.bar__big {
+  font-size: 16px;
+  height: 70px;
+  margin: 20px 0;
+  background: url(./images/Search.svg) no-repeat;
+  background-position: 26px;
+  background-size: 19px;
+  padding-left: 62px;
+  outline: none;
+}
+
+.bar__small {
+  font-size: 14px;
+  height: 42px;
+  background: url(./images/Search.svg) no-repeat;
+  background-position: 13px;
+  background-size: 11px;
+  padding-left: 33px;
+  outline: none;
+}
+
+.bar__big::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-size: 16px;
+  font-weight: 400;
+}
+
+.bar__small::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.bar__big:focus {
+  background: url(./images/Search.svg) no-repeat, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-position: 26px;
+  background-size: 19px;
+}
+
+.bar__small:focus {
+  background: url(./images/Search.svg) no-repeat, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-position: 13px;
+  background-size: 11px;
+}
+
+.bar:hover {
+  box-shadow: 0 3px 8px #3d4e6133;

```

Comment:
Also, you could use transition for hover,focus effects, but for now its not requeired